api/src/main/java/com/codeforcommunity/dto/site/AddSiteRequest.java

```diff
@@ -12,6 +12,7 @@ public class AddSiteRequest extends UpdateSiteRequest {
   private String zip;
   private String address;
   private Integer neighborhoodId;
+  private java.sql.Date plantingDate;
```

Comment:
:wrench: Since `plantingDate` is specific to a site entry (in other words, a site can have multiple site entries, each with a different planting date), could we move the `plantingDate` field to `UpdateSiteRequest`? Also, since we don't import anything else named `Date` here, we should just be able to do `import java.sql.Date;` at the top and do `private Date plantingDate`

Reply:
Agreed